src/pymoca/backends/casadi/mtensor.py

```diff
@@ -0,0 +1,29 @@
+import casadi as ca
+
+import numpy as np
+
+
+def new_mx(name, *shape):
```

Comment:
Does this method belong in this file, mtensor.py?

Have you considered setting up a general wrapper class that makes the decision whether to create a 'classical' MX or a flattened one internally? The class could be set up as a decorator for the underlying MX. We could call it 'TX', for example.

Reply:
I tried making it pretty with a wrapper class, but kept running into weird SWIG related stuff (overriding `__new__` did not work). So instead I opted to do it this way instead.

Reply:
Instead of subclassing, how about a simple wrapper class with an `mx` property?